randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -873,7 +913,13 @@ Extra - room_id: [31, 7]
   > Door to Data Room
       Trivial
   > Door to Data Save Room
-      Can Use Any Bombs
+      Any of the following:
+          Can Use Any Bombs
+          All of the following:
+              Morph Ball
+              Any of the following:
+                  Screw Attack
+                  Speed Booster and Movement (Intermediate)
```

Comment:
Morph blocks break from speedbooster? this should probably have knowledge tied to it as well.

Reply:
yup, so long as they're not chain blocks. But only if you collide into them. I don't think they break if you run across the top of them. Should that be Intermediate? That's my guess.